scripts/create_ai_entries.py

```diff
@@ -0,0 +1,233 @@
+            fh.write(message)
+    else:
+        sys.stderr.write(message)
+
+
+def create_entries(
+    s: requests.Session,
+    args: argparse.Namespace,
+) -> None:
+    """
+    Attempt to create entries for ai generated signatures.
+
+    :param s: requests session which is logged into pronto
+    :param args: cli args parser
+    """
+    submission_url = f"{args.pronto}/api/entry/"
+
+    for database in args.databases:
+        query_url = (
+            f"{args.pronto}/api/database/{database}/signatures/?integrated=0&commented=0"
+            f"&sort-by=reviewed-proteins&sort-order=desc&page_size={args.batch_size}"
+        )
+        sigs_query = s.get(query_url)
+
+        if sigs_query.status_code != 200:
+            logger.error(
+                "Failed to retrieve unintergated ai-generated signatures for %s:\n%s",
+                database,
+                sigs_query.text,
+            )
+            continue
+
+        sig_total, sig_count = len(sigs_query.json()['results']), 0
+        sig_perc = {f"{i*10}%": round(sig_total * i/10) for i in range(1, 11)}
+
+        for signature in sigs_query.json()['results']:
+            perc = (sig_count / sig_total * 100)
+            if perc in sig_perc.values():
+                logger.info("Parserd %s of signatures", sig_perc[perc])
+
+            sig_acc = signature['accession']
+            sig_type = signature['type']['code']
+            sig_url = f"{args.pronto}/api/signature/{sig_acc}/"
+
+            sig_response = s.get(sig_url)
+
+            if sig_response.status_code != 200:
+                logger.error("Could not retrieve data for signature %s", sig_acc)
+                continue
+
+            if is_human_complete(sig_response.json()):
+                # human curatored data is complete
+                # leave to curator to choose between human- and ai-generated data
+                continue  
+
+            if is_llm(sig_response.json()) is False:
+                continue
+
+            if sig_response.json()['llm_description'] is not None:
+                if len(sig_response.json()['llm_description']) > 30:
+                    logger.error(
+                        "Short name '%s' too long. Entry short names cannot be longer than 30 characters.\n",
+                        sig_response.json()['llm_description']
+                    )
+                    message = (
+                        f"SIGNATURE:{sig_acc}\t{sig_response.json()['llm_name']}\t{sig_response.json()['llm_description']}\t"
+                        f"ERROR: Short name too long.\n"
+                    )
+                    write_error(message, args)
+
+            payload = {
+                "type": sig_type,
+                "name": sig_response.json()['llm_name'],
+                "short_name": sig_response.json()['llm_description'],
+                'is_llm': True,
+                'is_llm_reviewed': False,
+                'is_checked': False,
+                'signatures': sig_acc,
```

Comment:
* We want the entry to be checked by default (otherwise it needs to be manually checked by curators to be propagated to the data warehouse for the release).
* `signatures` takes a _list_ of accessions.

```suggestion
                'is_checked': True,
                'signatures': [sig_acc],
```